### ISSUE TITLE
Sampling strategies & rare events

### ISSUE THREAD

**Issue by damian0604** [ccs-amsterdam/digitaltrackingworkshop18#7]
A major challenge in my experience is that even with large samples, most interesting behavior is very rare. I.e., most people do *not* visit a given, specific website on a given day, let alone read the same article. I'd like to discuss strategies to cope with and/or address this problem.

**Comment by kasperwelbers** [comment]
Hi @damian0604, would you be willing to host a roundtable on this? Perhaps combined with a brief presentation about the issues you encountered here with the desktop tracking project?

**Comment by magdalenawojcieszak** [comment]
I agree that this is a super important topic, thanks for proposing it!

**Comment by damian0604** [comment]
Yes, I can do that. Keep in mind I don't know the answer ;-)

**Comment by damian0604** [comment]
Some input for the discussion....
[rare_events_tracking_data.pdf](https://github.com/ccs-amsterdam/digitaltrackingworkshop18/files/2519712/rare_events_tracking_data.pdf)
